templates/CRM/PCP/Form/Search.tpl

```diff
@@ -0,0 +1,223 @@
+ | GNU Affero General Public License or the licensing of CiviCRM,     |
+ | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
+ +--------------------------------------------------------------------+
+*}
+<div class="crm-block crm-form-block crm-pcp-search-form-block">
+
+<h3>{ts}Find Campaign Pages{/ts}</h3>
+<table class="form-layout">
+  <tr>
+    <td>
+      {$form.title.label}<br />
+      {$form.title.html}<br />
+      <span class="description font-italic">
+          {ts}Complete OR partial pcp Name.{/ts}
+      </span>
+    </td>
+    <td>
+      {$form.supporter.label}<br />
+      {$form.supporter.html}<br />
+      <span class="description font-italic">
+          {ts}Complete OR partial pcp supporter Name/Email.{/ts}
+      </span>
+    </td>
+    <td>
+      {$form.status_id.label}<br />
+      {$form.status_id.html}<br />
+      <span class="description font-italic">
+          {ts}Filter search by Status.{/ts}
+      </span>
+    </td>
+  </tr>
+  <tr>
+    <td id="pcp_type-block">
+      {$form.page_type.label}<br />
+      {$form.page_type.html}<br />
+      <span class="description font-italic">
+          {ts}Filter search by pcp Type.{/ts}
+      </span>
+    </td>
+    <td>
+      {$form.page_id.label}<br />
+      {$form.page_id.html|crmAddClass:twenty}<br />
+      <span class="description font-italic">
+          {ts}Filter search by Contribution Page.{/ts}
+      </span>
+    </td>
+    <td>
+      {$form.event_id.label}<br />
+      {$form.event_id.html|crmAddClass:twenty}<br />
+      <span class="description font-italic">
+          {ts}Filter search by Event.{/ts}
+      </span>
+    </td>
+  </tr>
+  <tr>
+    <td>{$form.start_date.label}<br />
+      {include file="CRM/common/jcalendar.tpl" elementName=start_date}<br />
+      <span class="description font-italic">
+	{ts}Filter search by Contribution/Event Start Date{/ts}
+      </span>
+    </td>
+    <td>{$form.end_date.label}<br />
+      {include file="CRM/common/jcalendar.tpl" elementName=end_date}<br />
+      <span class="description font-italic">
+	{ts}Filter search by Contribution/Event End Date{/ts}
+      </span>
+    </td>
+  </tr>
+  <tr>
+     <td>{$form.buttons.html}</td><td colspan="2">
+  </tr>
+</table>
+</div>
+<table class="crm-pcp-selector">
+  <thead>
+    <tr>
+      <th class='crm-pcp-search-form-block-name'>{ts}Page Title{/ts}</th>
+      <th class='crm-pcp-supporter'>{ts}Supporter{/ts}</th>
+      <th class='crm-pcp-page-type'>{ts}Contribution Page/Event{/ts}</th>
+      <th class='crm-pcp-start'>{ts}Starts{/ts}</th>
+      <th class='crm-pcp-end'>{ts}Ends{/ts}</th>
+      <th class='crm-pcp-status'>{ts}Status{/ts}</th>
+      <th class='crm-pcp-pcp_links nosort'>&nbsp;</th>
+      <th class='hiddenElement'>&nbsp;</th>
+    </tr>
+  </thead>
+</table>
+
+{* handle enable/disable actions*}
+{include file="CRM/common/enableDisableApi.tpl"}
+
+{literal}
+<script type="text/javascript">
```

Comment:
So I guess the script can go - I'm not clear if there needs to be another include